modules/ROOT/pages/advanced/creating-custom-notifications.adoc

```diff
@@ -77,7 +77,7 @@ editor.notificationManager.open({
 });
 ----
 
-For a list of the icon identifiers, see: link:{baseurl}/advanced/editor-icon-identifiers/[Available icons].
+For a list of the icon identifiers, see: xref:advanced/editor-icon-identifiers.adoc[Available icons].
```

Comment:
xref:advanced/editor-icon-identifiers.adoc

should just be 
xref:editor-icon-identifiers.adoc

new docs has a flat file system, so most of the nesting will need to be removed.
That can happen in this PR or the next one :)

Reply:
@EkimChau you're mistaken here sorry, that only applies to the TinyMCE 6 docs.